content/docs/welcome-guide/requirements.md

```diff
@@ -119,3 +115,98 @@ If the current CMake version was not returned because CMake cannot be found, loc
+
+```shell
+wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | sudo tee /usr/share/keyrings/kitware-archive-keyring.gpg >/dev/null
+
+echo 'deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ focal main' | sudo tee /etc/apt/sources.list.d/kitware.list >/dev/null
+
+sudo apt-get update
+
+sudo apt-get install cmake
+```
+
+Once installed, verify that the version meets the minimum version criteria.
+
+```shell
+cmake --version
+```
+
+### Clang
+
+O3DE requires Clang to compile all of the native C++ code. Although Clang-6 is supported, it is recommended to use Clang-12 for the build process.
+
+```shell
+sudo apt-get install clang-12 
+```
+
+### Python 3.7 dependency on libffi
+
+Ubuntu 20.04.3 LTS does not support Python 3.7, however O3DE's local python package is Python 3.7 and has a library dependency on an earlier version of libffi that does not come with Ubunutu 20.04.3 LTS. You will need to manually install the older version of libffi onto Ubunutu 20.04.3 LTS in order for O3DE's Python to execute properly. The steps below demonstrate how to create a temp folder under `/tmp` to download and manually install the specific debian package.
+
+{{< note >}}
+Curl is required to perform the download step shown in the following instructions.
+{{< /note >}}
+
+
+```shell
+pushd /tmp
+
+LIBFFI_PACKAGE_NAME=libffi6_3.2.1-8_amd64.deb
+LIBFFI_PACKAGE_URL=http://mirrors.kernel.org/ubuntu/pool/main/libf/libffi/
+
+curl --location $LIBFFI_PACKAGE_URL/$LIBFFI_PACKAGE_NAME -o $LIBFFI_PACKAGE_NAME
+
+sudo apt install ./$LIBFFI_PACKAGE_NAME
+
+popd
+```
+
+### Additional Library Dependencies
+
+O3DE also requires some additional library packages to be installed.
+
+* libglu1-mesa-dev
+* libxcb-xinerama0
+* libxcb-xinput0
+* libxcb-xkb-dev
+* libxkbcommon-dev
+* libxkbcommon-x11-dev
+* libfontconfig1-dev
+* libcurl4-openssl-dev                    
+* libsdl2-dev                             
+* zlib1g-dev
+* mesa-common-dev
+
+These packages can be downloaded and installed through **apt-get**.
+
+```shell
+sudo apt-get install libglu1-mesa-dev libxcb-xinerama0 libxcb-xinput0 libfontconfig1-dev libcurl4-openssl-dev libsdl2-dev zlib1g-dev zlib1g-dev mesa-common-dev libxcb-xkb-dev libxkbcommon-dev libxkbcommon-x11-dev
+```
+
+### Ninja Build System (Optional)
+
+CMake supports Linux's default 'Unix Makefiles'. However, this does not support multiple configurations, so when a CMake project is generated, you must supply the configuration you want to build (debug, profile, or release). The Ninja build system, specifically [Ninja Multi-Config](https://cmake.org/cmake/help/latest/generator/Ninja%20Multi-Config.html), supports multiple configurations within the same generated build folder, so we recommend that you use this generator for O3DE development. Ninja build tool can be installed through **apt-get**:
```

Comment:
- It wasn't clear to me the relationship between Unix Makefiles and Ninja Multi-Config. I suggest rephrasing it to more clearly explain why they would need Ninja Build System, especially for those who aren't familiar. 
- Also, I'm not sure when you need to supply the configuration (debug, profile, or release)? Is this part necessary in this section?

```suggestion
CMake readily supports Linux's default Unix Makefiles; however, Unix Makefiles only supports single-configuration builds and not multi-configuration builds. The Ninja build system, specifically [Ninja Multi-Config](https://cmake.org/cmake/help/latest/generator/Ninja%20Multi-Config.html), supports multiple configurations within the same generated build folder. We recommend that you use this generator for O3DE development. You can install Ninja built tool through `apt-get`. 
```

Reply:
I will re-word this section to give more clarity